addons/l10n_pe_website_sale/demo/website_demo.xml

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo noupdate="1">
+
+    <record id="default_website_pe" model="website">
+        <field name="name">(PE) Peruvian Website</field>
+        <field name="company_id" ref="l10n_pe.demo_company_pe" />
+        <field name="logo" type="base64" file="l10n_pe_website_sale/static/description/icon.png" />
+        <field
+            name="domain"
+            model="ir.config_parameter"
+            eval="obj().env['ir.config_parameter'].sudo().get_param('web.base.url')"/>
+    </record>
+    <function model="payment.provider" name="copy">
+        <value eval="[ref('payment.payment_provider_transfer')]" />
+        <value eval="{'company_id': ref('l10n_pe.demo_company_pe'), 'state': 'enabled'}" />
+    </function>
```

Comment:
It doesn't work
![image](https://github.com/odoo/odoo/assets/60702485/b689c458-4861-4b62-b85c-06645763957e)
you need to publish it as well: `'is_published': True'`. I would also prefer if you use the "demo" provider

Reply:
Is it okay if I use transfer instead? the demo provider was asking for both its redirect_form_view_id and a journal_id, whilst wire transfer only needed its redirect_form_view_id.